### ISSUE TITLE
Error: SharedCredsLoad: failed to load shared credentials file caused by: FailedRead: unable to open file caused by: open /root/.aws/credentials: no such file or directory

### ISSUE THREAD

**Issue by bryanheo** [NetApp/terraform-provider-netapp-cloudmanager#111]
Hello

We are deploying NetApp CVO through Terraform Enterprise and AWS resources are created well but NetApp resources are not created due to the following credentials error. Terraform Enterprise does not store any credentials in file (eg /root/.aws/credentials) but netapp-cloudmanager_connector_aws resource requires SharedCredsLoad with /root/.aws/credentials.
Could you investigate the issue?

Provider
```
terraform {
  required_version = ">= 1.0.0"
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "~> 4.24"
 
    }
    
    netapp-cloudmanager = {
      source  = "netapp/netapp-cloudmanager"
      version = "~> 22.4.0"
    }
    local = {
      source  = "hashicorp/local"
      version = "~> 2.1"
    }
  }
}


provider "netapp-cloudmanager" {
  refresh_token = var.token
}

provider "aws" {
  region = var.primary_region
}
```
![Screenshot 2022-08-03 at 22 50 34](https://user-images.githubusercontent.com/12592663/182718887-348a9a31-ab78-4374-963a-a7852d1b3d99.png)

Error
```
Error: SharedCredsLoad: failed to load shared credentials file caused by: FailedRead: unable to open file caused by: open /root/.aws/credentials: no such file or directory
with module.use1.module.connector.netapp-cloudmanager_connector_aws.this[0]
on .terraform/modules/use1/modules/connector/connector.tf line 1, in resource "netapp-cloudmanager_connector_aws" "this":
```
![Screenshot 2022-08-02 at 19 28 37](https://user-images.githubusercontent.com/12592663/182717460-36aa23ab-fc8b-4382-a2d2-85fea54e4363.png)

Regards
Moon

**Comment by lonico** [comment]
I was able to recreate the same issue using Terraform Cloud with 22.4.0.

We just released 22.8.0, and it works for me without any change.   The logic related to AWS credentials was improved.
If this is not enough, you can now customise the path to the AWS profile file:
https://registry.terraform.io/providers/NetApp/netapp-cloudmanager/latest/docs#aws_profile_file_path

**Comment by nutanksingh** [comment]
I upgarded to the latest version for the netapp-cloudmanager provider today. My tf plan created the connector and it looks healthy by I get an panic error at the end of apply. Attaching the screenshot of the error and also the raw log.

<img width="1236" alt="Screen Shot 2022-08-04 at 4 56 36 PM" src="https://user-images.githubusercontent.com/8824315/182975556-5f66237f-ceba-4fb7-bebc-730f17b29d7b.png">
[netapp-cloudmanager-provider-22.8.0.txt](https://github.com/NetApp/terraform-provider-netapp-cloudmanager/files/9264223/netapp-cloudmanager-provider-22.8.0.txt)
.

**Comment by chuyich** [comment]
@bryanheo I'm able to create the aws connector with the latest version 22.8.0. I briefly check your provider.tf and suggest you set the provider version to 22.8.0 to see if that helps.
```
terraform {
  required_version = ">= 0.13"
  required_providers {
    netapp-cloudmanager = {
      source = "netapp/netapp-cloudmanager"
      version = "22.8.0"
    }
  }
}
```

**Comment by bryanheo** [comment]
@chuyich @lonico Thank you for your update.
With the new version, it works ok for me. @nutanksingh is my colleague and she is facing another issue with the new version as above .

**Comment by lonico** [comment]
fixed by 22.8.0.

I'm closing this isssue, as the new issue is tracked in https://github.com/NetApp/terraform-provider-netapp-cloudmanager/issues/112
